build.sbt

```diff
@@ -15,7 +15,7 @@ libraryDependencies ++= Seq(
   javaWs,
   javaJpa,
   "org.eclipse.persistence" % "eclipselink" % "2.6.2",
-  "com.github.ONSdigital" % "dp-dd-backend-model" % "develop-SNAPSHOT",
+  "com.github.ONSdigital" % "dp-dd-backend-model" % "feature~dao-SNAPSHOT",
```

Comment:
This pulls in the DAO from the feature/dao branch. Once that PR is merged, I will update this to use the develop/master version.